jupyter/minimal/ubi9-python-3.11/Dockerfile.cpu

```diff
@@ -55,12 +67,14 @@ RUN echo "Installing softwares and packages" && \
     jupyter labextension disable "@jupyterlab/apputils-extension:announcements" && \
     # Replace Notebook's launcher, "(ipykernel)" with Python's version 3.x.y \
     sed -i -e "s/Python.*/$(python --version | cut -d '.' -f-2)\",/" /opt/app-root/share/jupyter/kernels/python3/kernel.json && \
+    # copy jupyter configuration
```

Comment:
This part shouldn't be added also on the rest Dockfiles as well 🤔 ?

Reply:
That was Guillaume's improvement suggestion on top of everything else... Yeah, I guess we should have it in all images. True.

Reply:
@atheo89 it is done now, sorry the delay. I thought I had done it, but it was not there!